technic/helpers.lua

```diff
@@ -24,28 +24,28 @@ function technic.pretty_num(num)
 		if num == 0 then
 			-- handle 0 explicilty to avoid showing "-0"
 			if not constant_digit_count then
-				return "0 "
+				return "0 "
 			end
 			-- gives 0.000
-			return string.format("%.3f ", 0)
+			return string.format("%.3f ", 0)
 		end
-		return string.format("%.4g ", num)
+		return string.format("%.4g ", num)
 	end
 
 	num = num * 10 ^ (-3 * pref_i)
 	if constant_digit_count then
 		local comma_digits_cnt = 3 - (b - 3 * pref_i)
-		return string.format("%." .. comma_digits_cnt .. "f %s",
+		return string.format("%." .. comma_digits_cnt .. "f %s",
 			num, prefixes[pref_i])
 	end
-	return string.format("%.4g %s", num, prefixes[pref_i])
+	return string.format("%.4g %s", num, prefixes[pref_i])
 end
 
 -- some unittests
-assert(technic.pretty_num(-0) == "0 ")
-assert(technic.pretty_num(0) == "0 ")
-assert(technic.pretty_num(1234) == "1234 ")
-assert(technic.pretty_num(123456789) == "123.5 M")
+assert(technic.pretty_num(-0) == "0 ")
+assert(technic.pretty_num(0) == "0 ")
+assert(technic.pretty_num(1234) == "1234 ")
+assert(technic.pretty_num(123456789) == "123.5 M")
```

Comment:
I mean this should not be changed not even with extensive manual testing, nbsp and regular space are very different things and that's also why assertion tests were originally added.

Reply:
okay, sorry i will restore this.
I assumed that these were unintentional.

At this point too: https://github.com/mt-mods/technic/pull/255/files#diff-e385a425c8d367cc288be10c5cc80941966405fd86bf435a1b95a6206b74520fL657-R657 ?

Reply:
non-breaking spaces restored here https://github.com/mt-mods/technic/pull/255/commits/dbb7798f43b669f93e4b968dff2bd011ac27512c